quic/s2n-quic-core/src/packet/number/map.rs

```diff
@@ -769,4 +769,27 @@ mod tests {
             .with_type::<Vec<Operation>>()
             .for_each(|ops| model(ops))
     }
+
+
+    #[test]
+    #[cfg_attr(kani, kani::proof, kani::unwind(9))]
+    fn kani_test() {
+        // Confirm that a 
+        let gen = gen::<VarInt>();
+
+        check!()
+            .with_generator(gen)
```

Comment:
Thanks to type inference, this should be able to be simplified

```suggestion
        check!()
            .with_type()
```